packages/design-system/src/organisms/uniswapWidget/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { SwapWidget, SwapWidgetProps } from '@uniswap/widgets'
+import { useWebSocketProvider } from '../../hooks/web3New'
+import '@uniswap/widgets/fonts.css'
+
+// Default token list from Uniswap
+const UNISWAP_TOKEN_LIST = 'https://gateway.ipfs.io/ipns/tokens.uniswap.org'
+
+// Use the native token of the connected chain as the default input token
+const NATIVE = 'NATIVE' // Special address for native token
```

Comment:
Let's use a more explicit variable name to shorten the comments

```suggestion
const DEFAULT_INPUT_TOKEN_ADDRESS = 'NATIVE' // It's a magic keyword to use the native token of the chain you're connected to (ex. Ethereum -> ETH)
```